usaspending_api/references/tests/test_guide.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+
+from django.core.management import call_command
+
+@pytest.fixture()
```

Comment:
That's what I tried, but my `guide_data` fixture needs to use `db`, and the `db` fixture doesn't work with a module-scoped fixture:

`ScopeMismatch: You tried to access the 'function' scoped fixture 'db' with a 'module' scoped request object, involved factories`

... there might be some way around that (I'm trying to figure it out from http://pytest-django.readthedocs.io/en/latest/database.html - do you know one?)